modules/mfa/src/Auth/Process/Mfa.php

```diff
@@ -416,15 +418,16 @@ protected static function isHeadedToMfaSetupUrl($state, $mfaSetupUrl)
      *
      * @param int $mfaId The ID of the MFA option used.
      * @param string $employeeId The Employee ID that this MFA option belongs to.
-     * @param string $mfaSubmission The value of the MFA submission.
+     * @param string|array $mfaSubmission The value of the MFA submission.
      * @param array $state The array of state information.
      * @param bool $rememberMe Whether or not to set remember me cookies
      * @param LoggerInterface $logger A PSR-3 compatible logger.
      * @param string $mfaType The type of the MFA ('webauthn', 'totp', 'backupcode').
      * @param string $rpOrigin The Relying Party Origin (for WebAuthn)
      * @return string If validation fails, an error message to show to the
      *     end user will be returned.
-     * @throws \Sil\PhpEnv\EnvVarNotFoundException
+     * @throws EnvVarNotFoundException
+     * @throws Exception
```

Comment:
It's not letting me comment on the code 442-450. So this only needs one of those things defined? Because normally, I would expect all parameters to be provided for a function.

Reply:
No, I think that's just the style the original author decided to use to return only the first missing parameter in the error message. Obviously the `else`s aren't needed, but it wasn't flagged by lint, so I won't mess with it here.

Reply:
The problem is if mfaId is not empty, it doesn't check the rest.